src/App.js

```diff
@@ -20,6 +20,7 @@ import GeneralSearchContainer from './components/GeneralSearch/GeneralSearchCont
 import UserDashboard from './components/userDasboard/container/UserDashboard';
 import LoginDetails from './components/login/container/LoginDetails';
 import AddBulkConcepts from './components/bulkConcepts/addBulkConcepts';
+import BulkConceptPage from './components/bulkConcepts/container/BulkConceptsPage';
```

Comment:
I think this should be `CielConceptPage` as opposed to `BulkConceptPage` since that's a different page.

Reply:
The page will eventually render Bahmni concepts as well, so I think the name is correct. You may refer to the mockup for more clarity.

Reply:
So will the heading of the page